src/Microsoft.Win32.Primitives/src/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netcoreapp-Unix;
       netcoreapp-Windows_NT;
+      uap-Unix;
```

Comment:
That is actually a good point, I used a tool to add this configs so it added Unix configs wherever there was a netcoreapp-Unix one. @weshaggard I suppose I have to make all of these Windows-specific, is that right?